src/default/assets/js/src/typedoc/utils/transitions.ts

```diff
@@ -20,33 +20,38 @@ namespace typedoc {
     });
 
 
-    export function noTransition($el: JQuery, callback: () => void) {
-        $el.addClass('no-transition');
+    export function noTransition(el: HTMLElement, callback: () => void) {
+        el.classList.add('no-transition');
         callback();
-        $el.offset();
-        $el.removeClass('no-transition');
+        el.getBoundingClientRect();
```

Comment:
This doesn't do anything. Looks like the offset call earlier didn't either. Remove.

Reply:
I have check this, and animatition dosn't work without this call. I thought the same, that this doesn't do anything. But when I tried to test it I saw that it is broken. I spent about 15 mines to understand what happend. It looks like this call cause force update of styles and without it browser batchs all height changes together and we see final result without animation. May be I should write coment here, shell I? Or we can drop this animation. There is already fade in / fade out animation there. It is documetation generator, may be it make sence to simplify it.

Reply:
That's.... really surprising. I am okay with either dropping the animation or adding a comment, up to you. Thanks for checking!

Reply:
Height animation is removed (and styles for it too).